### ISSUE TITLE
Jest tests not working with serveStatic

### ISSUE THREAD

**Issue by lane711** [honojs/hono#1093]
I created a simple project with hono-minimal 

I then added
```
import { serveStatic } from "hono/cloudflare-workers";
app.use("/*", serveStatic({ root: "./" }));
app.use("/favicon.ico", serveStatic({ path: "./favicon.ico" }));
```

now when I run `npm run test` I get this error:
```
    Cannot find module '__STATIC_CONTENT_MANIFEST' from 'node_modules/hono/dist/cjs/adapter/cloudflare-workers/server-static-module.js'

    Require stack:
      node_modules/hono/dist/cjs/adapter/cloudflare-workers/server-static-module.js
      node_modules/hono/dist/cjs/adapter/cloudflare-workers/index.js
      src/index.ts
      src/index.test.ts
```

https://github.com/lane711/hono-minimal/blob/main/src/index.ts

**Comment by yusukebe** [comment]
Hi @lane711 !

You need to add a stub for `__STATIC_CONTENT_MANIFEST` by referring to the following:

https://github.com/kriasoft/cloudflare-starter-kit/pull/22

**Comment by yusukebe** [comment]
Related to #1127

**Comment by yusukebe** [comment]
Resolved by #1984
